### PR TITLE
00 sort bi extracts

### PR DESCRIPTION
**Pull request by lizihao-anu**
Use fs::file_copy to replace file.copy to improve the speed

### CODE REVIEW

**Review by Jennit07 — COMMENTED** [review]
```
[1] "source-episode-file-2223.parquet"    "source-individual-file-2223.parquet"
Error: [ENOENT] Failed to copy '/conf/sourcedev/Source_Linkage_File_Updates/2223/source-episode-file-2223.parquet' to '/conf/hscdiip/copy_test/source-episode-file-2223.parquet': no such file or directory
```

When trying to test one year, i get this error even though i created a `copy_test` folder in hscdiip

File: copy_to_hscdiip.R [hunk 1]
```diff
@@ -0,0 +1,32 @@
+dir_folder <- "/conf/sourcedev/Source_Linkage_File_Updates"
+target_folder <- "/conf/hscdiip/copy_test"
```
Comment (Jennit07) [inline copy_to_hscdiip.R:2]:
```suggestion
target_folder <- "/conf/hscdiip/"
```
When this is ready for merging back in, we need to remember to remove the test folder

**Review by SwiftySalmon — COMMENTED** [review]
Look good! Will test it tomorrow and see if I get any permission issues

**Review by SwiftySalmon — APPROVED** [review]
All seems good. We can test properly when we get the next boxi extracts

**Pull request merged**